CODEOWNERS

```diff
@@ -3,3 +3,4 @@
 
 /pre-docs @cwcowellshah @puppetlabs/bolt
 /lib/bolt_server @puppetlabs/skeletor
+CHANGELOG.md @cwcowellshah @puppetlabs/bolt
```

Comment:
@cwcowellshah caught this problem. I think that he should be auto-requested for review for CHANGELOG.md PRs